packages/web-components/src/components/cbp-toast/cbp-toast.tsx

```diff
@@ -0,0 +1,63 @@
+import { Component, Prop, Host, Watch, h } from '@stencil/core';
+
+@Component({
+  tag: 'cbp-toast',
+  styleUrl: 'cbp-toast.scss'
+})
+export class CbpToast {
+
+  /** specifies the icon loaded into the sidebar */
+  @Prop() icon: string = 'user';
+
+  /** specifies the color for the toast */
+  @Prop({ reflect: true }) color: 'info' | 'danger' | 'success' | 'warning' = 'info';
+
+  /** specifies time in seconds for the toast to be displayed */
+  @Prop() timer: 3 | 5 | 10;
+
+  /** When set, specifies that the toast is open */
+  @Prop({ reflect: true }) open: boolean = true;
+
+  /** Specifies the context of the component as it applies to the visual design and whether it inverts when light/dark mode is toggled. Default behavior is "light-inverts" and does not have to be specified. */
+  @Prop({ reflect: true }) context: "light-inverts" | "light-always" | "dark-inverts" | "dark-always";
+  
+  /** Supports adding inline styles as an object */
+  @Prop() sx: any = {};
+
+  @Watch('open')
+    watchOpenHandler(newValue: boolean){
+      console.log('watchOpenHandler check');
+      if(!newValue) {
+        console.log('dismiss toast!');
+      }
+    }
+
+  render() {
+
+    if(this.open && this.timer){
+      setTimeout(() => { this.open = false }, this.timer * 1000)
+    }
+    
+    return (
+      <Host>
+        <div class='cbp-toast-sidebar'>
+          <cbp-icon
```

Comment:
This locks them in to our icons. I would just provide a named slot so they can slot an icon component or include an external icon if needed.